index.mjs

```diff
@@ -64,55 +64,31 @@ discordClient.on('ready', () => {
     console.log(message);
 
     discordClient.users.fetch(process.env.ADMIN_ID, false)
-        .then((user) => {
+        .then(user => {
             user.send(message);
         });
 
-    discordClient.user.setActivity('tarkov.dev', {
+    discordClient.user.setActivity('Tarkov.dev', {
         type: 'PLAYING',
     });
 });
 
 discordClient.login(process.env.DISCORD_API_TOKEN);
 
-discordClient.on('guildCreate', async (guild) => {
+discordClient.on('guildCreate', async guild => {
     if (!guild.available) {
         return false;
     }
 
-    const message = `Joined server ${guild.name} with ${guild.memberCount} members (${guild.id})!`;
-
-    console.log(message);
-
-    discordClient.users.fetch(process.env.ADMIN_ID, false)
-        .then((user) => {
-            user.send(message);
-        });
-
     try {
         const owner = await guild.fetchOwner();
-        owner.send(`Thank you so much for adding the Stash bot to your Discord!\n\rTo get more information on how the bot works, try /help to get started.`);
-    } catch (someError) {
-        console.error(someError);
-    }
-});
-
-discordClient.on('guildDelete', (guild) => {
-    if (!guild.available) {
-        return false;
+        owner.send(`Thank you so much for adding the Stash bot to your Discord!\n\rTo get more information on how the bot works, try \`/help\` to get started.`);
+    } catch (error) {
+        console.error(error);
     }
-
-    const message = `Kicked from server ${guild.name} with ${guild.memberCount} members (${guild.id})!`;
-
-    console.log(message);
-
-    discordClient.users.fetch(process.env.ADMIN_ID, false)
-        .then((user) => {
-            user.send(message);
-        });
```

Comment:
This section is pretty helpful for the bot admins to see if the bot gets kicked from a server